include/math/QDiv.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
```

Comment:
Bold assumption that you know better how to optimize division that your compiler, do you have any test for that?

Reply:
It is how compilers optimize division/modulo by powers of 2
https://godbolt.org/z/Wo7694ezj

Reply:
Key word: compilers optimize, I think we don't need it in our code, compiler will do it better